encrypted-media-respec.html

```diff
@@ -3911,13 +3911,16 @@
             The resulting identifier(s) MUST be <a def-id="non-associable-by-application"></a> and <a href="#uses-distinctive-identifiers">use of them</a> MUST only be exposed to a <a href="#per-origin-identifiers">single origin</a>.
             This process MAY be performed multiple times, such as after identifier(s) are <a href="#allow-identifiers-cleared">cleared</a>.
           </p>
-          <p>This process MUST be performed either <a href="#direct-individualization">directly by the user agent</a> or <a href="#app-assisted-individualization">through the application</a>.
+          <p>This process MUST be performed either <a href="#direct-individualization">directly by the user agent</a> or <a href="#app-assisted-individualization">through the application</a> depending on the CDM implementation.
             The mechanisms, flow, and restrictions for the two types of individualization are different, as described in the following sections.
           </p>
 
           <section id="direct-individualization">
             <h5>Direct Individualization</h5>
-            <p>Direct Individualization is performed between the <a def-id="cdm"></a> and an origin- and application-independent server.
+            <p>
+              Direct Individualization is performed between the <a def-id="cdm"></a> and an origin- and application-independent server.
+              Although the server is origin-independent, the result of the individualization enables the CDM to provide origin-specific identifiers per the other
+              requirements of this specification.
```

Comment:
I noticed a typo at 3936: I believe “origin(s)-” in the last paragraph of this section is _not_ supposed to have a dash. The dash in “orign-” _is_ correct.